.jscs.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
It's addressed in line 27 of workflow.rst https://github.com/nengo/nengo_gui/pull/605/files#diff-ff11f4d5436e9c5c3e9fdb37dd16c379R27